src/main/jbake/content/community/docs/developer-guide/alerts.adoc

```diff
@@ -52,6 +52,11 @@ There are several different kinds of conditions but they all have one thing in c
 ** X > 10 per-minute
 * Availability
 ** X is DOWN
+* Event
+** event.id starts 'IDXYZ', event.tag.category == 'Server',
+   event.tag.from ends '.com'
+* Missing
+** Not X in last 5 minutes
```

Comment:
The same X used in the examples:
X < 10, X >= 20
X inside [10,20), X outside [100,200]
X < 80% Y
X starts with "ABC", X matches "A.*B"
X > 10 per-minute
X is DOWN

Which means "not event/data received for X" in the last 5 minutes.